lang/java/reef-webserver/src/main/java/org/apache/reef/webserver/HttpServerReefEventHandler.java

```diff
@@ -189,8 +193,9 @@ public void onHttpRequest(
         }
       }
       break;
-    // TODO[JIRA REEF-798] Use this provider in the HTTP
     case "progress":
+      response.getWriter().println(progressProvider.getProgress());
```

Comment:
Should we return a JSON object rather than simply just a number? Or does a single number suffice?